schema/alpha/execution.json

```diff
@@ -0,0 +1,43 @@
+{
+    "$id": "http://github.com/kubeflow/metadata/schema/alpha/execution.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "description": "a run of an executable",
+    "allOf": [
+        {
+            "$ref": "http://github.com/kubeflow/metadata/schema/alpha/entity.json"
+        },
+        {
+            "properties": {
+                "category": {
+                    "constant": "execution"
+                },
+                "executable_id": {
+                    "type": "string",
+                    "description": "the id of the executable"
+                },
+                "configuration": {
```

Comment:
Is this supposed to be a catch all? It's great for flexibility but harder to operate on. Also brings up the question as to whether other entities should have a configuration property

Reply:
replied inline